dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/ApplicationProcessorTest.java

```diff
@@ -144,6 +144,10 @@ public void testApplicationProcessor() throws Exception {
         assertEquals("FAIL: Only a single application must be processed by the application processor.", app1Doc,
             openapi);
 
+        // Check we didn't output the beta message
```

Comment:
strictly not a beta-only message

Reply:
Not quite sure what you're getting at. Outputting the message should be gated behind a beta fence.

Reply:
I just mean that at some point the beta guard will be removed and this message will be output from GA code given the appropriate config options

Reply:
Ah sure, but at that point we'll need to remove this part of the test anyway since it'll start failing.